cmd/collector/app/span_processor.go

```diff
@@ -159,6 +161,11 @@ func (sp *spanProcessor) enqueueSpan(span *model.Span, originalFormat SpanFormat
 	//add format tag
 	span.Tags = append(span.Tags, model.String("internal.span.format", string(originalFormat)))
 
+	// append the collector tags
+	for k, v := range sp.collectorTags {
```

Comment:
add `// TODO add support for deduping tags, https://github.com/jaegertracing/jaeger/issues/1778`

Reply:
Done.